tests/stub/bookmarks/test_bookmarks_v4.py

```diff
@@ -55,7 +54,8 @@ def test_last_bookmark(self):
 
     def test_send_and_receive_bookmarks_read_tx(self):
         self._server.start(
-            path=self.script_path("send_and_receive_bookmark_read_tx.script")
+            path=self.script_path(self.scripts_path,
```

Comment:
Nitpicky me: this naming is a bit confusing. Maybe rename `self.scripts_path` to something like `self.version_folder` or whatever. Or just ignore that comment, it's not a deal-breaker.

Reply:
Personally, the chosen name was fine to me, but I do not mind updating it.